common/util.go

```diff
@@ -0,0 +1,14 @@
+package common
+
+import "time"
+
+func WaitUntil(waitTime time.Duration, condition func() bool) {
+	start := time.Now()
+	for time.Since(start) < waitTime {
+		if condition() == false {
+			time.Sleep(100 * time.Millisecond)
```

Comment:
Any reason in particular for using this `time.Sleep` pattern rather than the golang Timer pattern with channels? More just curious...not suggesting otherwise.

Reply:
I wanted to periodically check if a condition is true, and have a max total wait-time.  I think Timer only allows you to wait for the whole period right?